### ISSUE TITLE
Completions are not accurate in qualifed identifier context

### ISSUE THREAD

**Issue by malinthar** [ballerina-platform/ballerina-lang#34753]
**Description:**
As depicted below, there are two bal files in the same module. One has an import statement which imports the `io` module.

Case 1
![completion-issue](https://user-images.githubusercontent.com/35211477/150338206-9fddf2c0-690b-4e71-90ad-9ddc464b8b49.gif)

The completions in the `io:<cursor>` context is not accurate as the `io` module is not imported in the current bal file.

Case 2
![completion-issue2](https://user-images.githubusercontent.com/35211477/150338793-af852b18-2623-4bda-8495-00c32b39ec67.gif)

When I select the completion item suggested for importing the `io` module it adds an alias in the import statement though there is no clashing module names in the file. Moreover, no completions are provided for qualified name reference context as depicted above. 

**Steps to reproduce:**
Create a module in a ballerina project. And add the following files. 

file 1
```ballerina
import ballerina/io;

public function main() {
        io:print("Hello")
}
```
file2

```ballerina
function test() {
    io:<cursor>
}
```

**Affected Versions:**
SL-GA-rc1

**Comment by malinthar** [comment]
The reason for case 1 seems to be due to the behavior of the visible symbols API. It returns all the symbols that belong to a module instead of the file.

**Comment by pubudu91** [comment]
Closing since this is fixed with #34791
